package.json

```diff
@@ -22,11 +22,13 @@
     "@types/node": "12.12.53",
     "@types/react": "16.9.43",
     "@types/react-dom": "16.9.8",
+    "cross-env": "^7.0.2",
     "cypress": "4.11.0",
     "dependency-cruiser": "9.9.3",
     "eslint-config-prettier": "6.11.0",
     "eslint-config-react-app": "5.2.1",
     "eslint-plugin-cypress": "2.11.1",
+    "eslint-plugin-import": "^2.22.0",
```

Comment:
Did this dependency end up providing any benefit?

Reply:
Ah no, I think I can remove it. Good call. 👍

Reply:
When I remove this eslint doesn't work any more. Which is weird because it worked before I added this explicit dependency. 😓

Reply:
Have you done a post uninstall `npm ci`?

Reply:
Fixed in 09939c8